docs/infra/Infra-Ticket-Queue.md

```diff
@@ -14,10 +14,10 @@ This allows the team to separate their engineering work from ["toil" work](https
 
 IMPORTANT: Whenever you have a request for the infra team, please file a ticket instead of contacting team members directly, even for seemingly trivial things or even if an individual has done the same thing for you in the past. Infra on-call will be there to handle your request, and it lets non on-call team members focus on their engineering tasks.
 
-When in doubt, ask on the `#hackers-infra` channel in [[Chat]].
+When in doubt, ask on the `#hackers-infra` channel in [Chat](https://github.com/flutter/flutter/wiki/Chat).
 
 # How to File a Ticket as an Infra Customer
-1. Open a [new infra issue](https://github.com/flutter/flutter/issues/new?assignees=&labels=team-infra&projects=&template=6_infrastructure.yml). (That template summarizes the information on this page.)
```

Comment:
@goderbauer did we dissolve the `team-infra` label? I have it on my todo list to triage it weekly, but I'm open to other processes.

Reply:
The `team-infra` label still exists and is supposed to be triaged. We only decommissioned the `team-google-testing` label. I am happy to work with you on figuring out something else for team-infra if the current process isn't working, though.

Reply:
Totally up for changing it in a later PR, right now I am just moving docs around quickly, trying to not get caught up in changing the content. 🙂  🙏

Reply:
I *think* for now we should go with it, we'll see how often these end up in critical issue triage :)

@piinks should we leave the `labels=team-infra` in this link, or was there another reason to remove it?

Reply:
team-infra is already applied automatically a part of the template: https://github.com/flutter/flutter/blob/3bb9624ca983709adc793c36e88d0f4a86607a35/.github/ISSUE_TEMPLATE/6_infrastructure.yml?plain=1#L6

The analyze.dart script enforces only permitted `file an issue` links. Which TIL. 😅

Reply:
Ahh, cool, TIL too